$OpenBSD: patch-WindowMaker_background_menu,v 1.1 2017/09/08 13:46:33 dcoppa Exp $

commit 273d17a0be1a00435e3b98791359b47908dab443
Author: Doug Torrance <dtorrance@piedmont.edu>
Date:   Fri Aug 18 01:02:24 2017

menu: Include all aspect ratio options for background pixmap.

The background menu was missing "centered", "maximized", and "filled".

This closes Debian bug #85591 [1]:
  The current default for the scaled backgrounds menu is to use
  scaling without keeping the aspect ratio.

  This very rarely yields to a usable background.

  Could the default be changed to scaling with keeping the aspect
  ratio ?

[1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=85591

Index: WindowMaker/background.menu
--- WindowMaker/background.menu.orig
+++ WindowMaker/background.menu
@@ -24,5 +24,8 @@
         "Images" MENU
             "Tiled" OPEN_MENU LOCAL_BACKGROUNDS_DIR BACKGROUNDS_DIR USER_BACKGROUNDS_DIR WITH wmsetbg -u -t
             "Scaled" OPEN_MENU LOCAL_BACKGROUNDS_DIR BACKGROUNDS_DIR USER_BACKGROUNDS_DIR WITH wmsetbg -u -s
+            "Centered" OPEN_MENU LOCAL_BACKGROUNDS_DIR BACKGROUNDS_DIR USER_BACKGROUNDS_DIR WITH wmsetbg -u -e
+            "Maximized" OPEN_MENU LOCAL_BACKGROUNDS_DIR BACKGROUNDS_DIR USER_BACKGROUNDS_DIR WITH wmsetbg -u -a
+            "Filled" OPEN_MENU LOCAL_BACKGROUNDS_DIR BACKGROUNDS_DIR USER_BACKGROUNDS_DIR WITH wmsetbg -u -f
         "Image" END
 "Background" END
